lib/misc.js

```diff
@@ -65,56 +65,71 @@ exports.generateId = function (fields) {
   return base32hex.encodeBuffer(hasher.digest(), { paddingChar: '-' });
 };
 
-var userIdsConfig = function (env, serverSecret) {
-  const key = crypto.createHash('sha256').update(serverSecret).digest();
-
+var userIdsPath = function (env) {
   return {
-    encryptedFilePath: `${__dirname}/platform_users/${env}.json.enc`,
-    unencryptedFilePath: `${__dirname}/platform_users/${env}.json`,
-    ivBlockSize: 16,
-    key: key,
-    algorithm: 'aes-256-cbc',
+    hashedPath: `${__dirname}/platform_users/${env}_hashed.json`,
+    unhashedPath: `${__dirname}/platform_users/${env}.json`,
```

Comment:
Consider just removing the unhashed path and allow it as a parameter on `hashUserIds`. Its location is no longer relevant since the ids are being appended to the existing hashed ids. (Before, with encrypting and decrypting we had to keep the files associated.)